module_report/check_modules.py

```diff
@@ -0,0 +1,53 @@
+import importlib
+from pathlib import Path
+import platform
+import subprocess
+
+
+lib_path = Path([*Path('.venv').rglob('site-packages')][0])
+libs = ['PyQt5', 'PyQt6', 'PySide2', 'PySide6']
+qtpy_files = [f.name[:-len('.py')] for f in Path('qtpy').glob('*.py') if f.name.startswith('Qt')]
+
+
+modules = {}
+
+
```

Comment:
```suggestion
FALSE_POSITIVES = [
    'QtChart',
    'QtOpenGLFunctions',
    'QtScript',
    'QtScriptTools',
]


```

Add false positives here so the check can be performed with no unnecessary complexity